packages/relay/src/index.ts

```diff
@@ -62,31 +63,31 @@ export interface Net {
 }
 
 export interface Eth {
-  blockNumber(requestId?: string): Promise<string>;
+  blockNumber(requestIdPrefix: string): Promise<string>;
```

Comment:
I think it would be better to make the code base consistent and use `requestDetails` everywhere even for methods which would only use the `requestId` field of `requestDetails`.

Also, let's make it optional for methods which are used outside of the server context, e.g. in `poller.ts`

Reply:
If i make it optional for the methods in poller e.g getLogs, this will affect the whole relay, not sure if its a good idea, isnt it better to just handle the passing of requestId in the poller as I mentioned above